conf/test_full.config

```diff
@@ -16,6 +16,7 @@ params {
 
   // Genome references
   genome = 'hg19'
```

Comment:
```suggestion

```

Reply:
Or this one.

Reply:
Will need to add links to Kraken databases here when we upload them to Zenodo too.